modules/administration-guide/pages/enabling-users-to-run-multiple-workspaces-simultaneously.adoc

```diff
@@ -0,0 +1,42 @@
+:_content-type: PROCEDURE
+:navtitle: Enabling users to run multiple workspaces simultaneously
+:description: Enabling users to run multiple workspaces simultaneously
+:keywords: administration guide, number, workspaces
+
+[id="enabling-users-to-run-multiple-workspaces-simultaneously_{context}"]
+= Enabling users to run multiple workspaces simultaneously
+
+By default, a user can run only one workspace at a time. You can enable users to run multiple workspaces simultaneously.
```

Comment:
This was discussed today in the sprint retro, and I've realized now that we should add some caveats here about pods, nodes, and storage types.

```
NOTE: If using the `common` storage method, users may experience problems with concurrently running workspaces if pods are distributed across nodes in the cluster. Switching to `per-user` storage, or using the `ephemeral` storage type, can help with this. 
```

cc: @amisevsk @ibuziuk Does that sound reasonable/illogical?

Reply:
Makes sense to me, though I'm not sure we've documented a good way of selecting `ephemeral` or `per-workspace`.

Also, `s/per-user/per-workspace/` -- per-user is an alias for `common`, which is the default.